gene/etl/base.py

```diff
@@ -150,3 +154,108 @@ def get_seqrepo(self, seqrepo_dir: Path) -> SeqRepo:
+            location["end"] = start
+
+    # Add back once VRS Chromosome Location is supported in 2.0-alpha
+    # def _get_chromosome_location(self, location: Dict, gene: Dict) -> Optional[Dict]:
+    #     """Transform a gene's location into a GeneChromosomeLocation.
+
+    #     :param location: A gene's location.
+    #     :param gene: A transformed gene record.
+    #     :return: If location is a valid VRS ChromosomeLocation, return a dictionary
+    #         containing the ChromosomeLocation. Else, return None.
+    #     """
+    #     if "chr" in location and "start" in location and "end" in location:
+    #         if location["start"] == "p" and location["end"] == "p":
+    #             location["start"] = "pter"
+    #             location["end"] = "cen"
+    #         elif location["start"] == "q" and location["end"] == "q":
+    #             location["start"] = "cen"
+    #             location["end"] = "qter"
+    #         try:
+    #             chr_location = GeneChromosomeLocation(
+    #                 chr=location["chr"],
+    #                 start=location["start"],
+    #                 end=location["end"]).model_dump()
+    #         except ValidationError as e:
+    #             logger.info(f"{e} for {gene['symbol']}")
+    #         else:
+    #             return chr_location
+    #     return None
+
+    def _get_seq_id_aliases(self, sr: SeqRepo, seq_id: str) -> List[str]:
+        """Get GA4GH aliases for a sequence id
+
+        :param sr: seqrepo instance
+        :param seq_id: Sequence ID accession
+        :return: List of aliases for seqid
+        """
+        aliases = []
+        try:
+            aliases = sr.translate_alias(seq_id, target_namespaces="ga4gh")
+        except KeyError as e:
+            logger.warning(f"SeqRepo raised KeyError: {e}")
+        return aliases
+
+    def _get_sequence_location(
+        self, seq_id: str, gene: Feature, params: Dict, sr: SeqRepo
+    ) -> Dict:
+        """Get a gene's GeneSequenceLocation.
+
+        :param seq_id: The sequence ID.
+        :param gene: A gene from the source file.
+        :param params: The transformed gene record.
+        :param sr: Access to the SeqRepo
+        :return: A dictionary of a GA4GH VRS SequenceLocation, if seq_id alias found.
+            Else, empty dictionary
+        """
+        location = {}
+        aliases = self._get_seq_id_aliases(sr, seq_id)
+        if not aliases:
+            return location
+
+        sequence = aliases[0]
+
+        if gene.start != "." and gene.end != "." and sequence:
+            if 0 <= gene.start <= gene.end:  # type: ignore
+                location = GeneSequenceLocation(
+                    start=gene.start - 1,  # type: ignore
+                    end=gene.end,  # type: ignore
+                    sequence_id=sequence
+                ).model_dump()  # type: ignore
+            else:
+                logger.info(f"{params['concept_id']} has invalid interval:"
```

Comment:
This might be closer to `logger.warning` than `logger.info`

Reply:
This PR just focused on moving the code from the classes. I didn't look to modifying code in the methods, but can make this change here since it's a small one